// Họ tên: Đoàn Minh Thịnh - MSSV: 25810040
fun main() {
    val canNang: Double = 69.5
    val chieuCao: Double = 1.74
    
    val bmi: Double = canNang / (chieuCao * chieuCao)

    val phanLoai: String
    if (bmi < 18.5) {
        phanLoai = "Gầy"
    } else if (bmi < 25.0) {
        phanLoai = "Bình thường"
    } else if (bmi < 30.0) {
        phanLoai = "Thừa cân"
    } else {
        phanLoai = "Béo phì"
    }

    println("Chiều cao: $chieuCao m")
    println("Cân nặng: $canNang kg")
    println("Chỉ số BMI: $bmi")
    println("Phân loại sức khỏe: $phanLoai")
}
